sa/database.go

```diff
@@ -182,3 +183,10 @@ func initTables(dbMap *gorp.DbMap) {
 	dbMap.AddTableWithName(core.SignedCertificateTimestamp{}, "sctReceipts").SetKeys(true, "ID").SetVersionCol("LockCol")
 	dbMap.AddTableWithName(core.FQDNSet{}, "fqdnSets").SetKeys(true, "ID")
 }
+
+func safeSelectOne(m *gorp.DbMap, holder interface{}, q string, args ...interface{}) error {
+	if strings.ContainsAny(q, "'\"\n\r\\") {
+		return errors.New("Query contains unsafe character")
+	}
+	return m.SelectOne(holder, q, args...)
```

Comment:
Thinking out loud: I notice all of the callers of `safeSelectOne` are using a `q` value like: `fmt.Sprintf("SELECT %s FROM ...", fieldList)`. Do you think there's any value in adding another wrapper on top of `safeSelectOne` that takes a fieldlist and produces the boilerplate SELECT prefix with `sprintf` or is that overkill?